cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -0,0 +1,244 @@
+        }) }}
+    </div>
+    <div class="block">
+        {{ post_preview.render({
+            'heading': 'This is a blog post',
+            'body': lipsumText,
+            'published_date': '20030805T213611',
+            'image_url': 'http://www.placehold.it/160x90',
+            'post_category': ['Info for Consumers', 'At the CFPB'],
+            'author': ['Tom', 'Jerry'],
+            'post_tags': ['FOIA', 'Mortgage']
+            },
+            '/blog/') }}
+    </div>
+    <div class="block
+                block__flush-top">
+        {{ full_width_text.render( 'sublanding-page/_template-prototype-full-width-content.html' ) }}
+    </div>
+    <div class="block">
+        {{ table.render( [
+                            'Column 1', 'Column 2', 'Column 3'
+                          ],
+                          [
+                            ['Cell A1', 'Cell A2', 'Cell A3'],
+                            ['Cell B1', 'Cell B2', 'Cell B3'],
+                            ['Cell C1', 'Cell C2', 'Cell C3']
+                          ]
+        ) }}
+    </div>
+    <div class="block">
+        {{ well.render( 'landing-page/_template-prototype-well-content.html' ) }}
+    </div>
+    <div class="o-link-blob-group">
+        {{ link_blob.render({
+            'heading': 'This is an example of a Half Width Link Blob',
+            'body':    'To highlight navigational links and provide
+                        context about what users can expect on the linked
+                        page',
+            'links': [
+                {
+                    'text': 'This is a Link List molecule inside the blob',
+                    'url':  '/some-url/'
+                },
+                {
+                    'text': 'This is another Link List molecule inside the
+                             blob',
+                    'url':  '/some-other-url/'
+                }
+            ]
+        }) }}
+        {{ link_blob.render({
+            'heading': 'This is another example of a Half Width Link Blob',
+            'body':    '<strong>Example:</strong> About Us landing page,
+                        sub-page descriptors on "office" pages and
+                        wireframed on Rulemaking landing page.',
+            'links': [
+                {
+                    'text': 'This is a Link List molecule inside the blob',
+                    'url':  '/some-url/'
+                }
+            ]
+        }) }}
+    </div>
+    <div class="block
+                block__flush-top">
+        {{ main_contact_info.render( {
+            'heading': 'Whistleblower tips',
+            'body':    lipsumText
+        }, {
+            'emails':  ['test@example.com'],
+            'phones':  [{'number': '1234567899',
+                         'vanity': '123456AAAA',
+                         'tty':    '1234567899'}]
+        }) }}
+    </div>
+{% endblock %}
+
+{% block content_sidebar scoped -%}
+    {{ related_posts.render({
```

Comment:
This is included in the breakout sidebar organism above. Does it still need to be here?